packages/accounts/package.json

```diff
@@ -8,6 +9,8 @@
     "start": "vite"
   },
   "dependencies": {
+    "@sentry/react": "^6.5.1",
+    "@sentry/tracing": "^6.5.1",
```

Comment:
Please use exact versioning and remove `^`. Not sure why [.npmrc](https://github.com/pingcap/tug-website/blob/main/packages/accounts/.npmrc#L2) did not work properly for you..